configureoptimization.cmake

```diff
@@ -17,6 +17,14 @@ elseif(CLR_CMAKE_PLATFORM_UNIX)
         add_compile_options($<$<CONFIG:Checked>:-O2>)
         add_compile_options($<$<CONFIG:Release>:-O3>)
         add_compile_options($<$<CONFIG:RelWithDebInfo>:-O2>)
+
+        if (CLR_CMAKE_BUILD_COREBUNDLE)
+            add_compile_options($<$<CONFIG:Release>:-Os>)
```

Comment:
I would prefer adding the optimization option just once for release builds. With this change, it will pass both `-O3` and `-Os`. I assume the compiler will use the last one it can see, but it is a bit grey zone.